frontend/src/api/captions.ts

```diff
@@ -0,0 +1,35 @@
+import axios, { AxiosResponse } from 'axios';
+import { API_URL } from '../utils/constants';
+
+export type EnableCaptionsType = {
+  captionsId: string;
```

Comment:
Do we ever have a `message` property?

Reply:
any updates, @behei-vonage ?

Reply:
same applies for the `DisableCaptionsType`. :pray: